baselines/ts5.5/dom.generated.d.ts

```diff
@@ -36690,7 +36690,7 @@ interface WebSocket extends EventTarget {
      *
      * [MDN Reference](https://developer.mozilla.org/docs/Web/API/WebSocket/readyState)
      */
-    readonly readyState: number;
+    readonly readyState: typeof WebSocket.CONNECTING | typeof WebSocket.OPEN | typeof WebSocket.CLOSING | typeof WebSocket.CLOSED;
```

Comment:
```suggestion
    readonly readyState: 0 | 1 | 2 | 3;
```
This is more performant and compiles to the same output. Using the WebSocket.CONNECTING | ... approach adds unnecessary complexity without functional benefit.

Reply:
I tend to agree with @Bashamega.